package.json

```diff
@@ -53,5 +53,8 @@
     "turbo": "^2.3.3",
     "typedoc": "^0.27.2",
     "typescript": "^5.4.5"
+  },
+  "dependencies": {
+    "opensea-js": "^7.1.15"
```

Comment:
Please move this to [typescript/agentkit/package.json](https://github.com/coinbase/agentkit/blob/master/typescript/agentkit/package.json#L41-L48)